### PR TITLE
Feature/better retail styling update

### PR DESCRIPTION
**Pull request by a-rousseau**
[AB#66620](https://dev.azure.com/orckestra001/da965986-c85b-43be-9915-219568b000e5/_workitems/edit/66620)
Fix alignment of icons in mobile
Adjust the padding of the navbar on mobile
Fix styling of the jumbotron heading font weight
Fix the header spacing by hiding the div with hidden content
Fix the row gap to take into account the product tile banners
Fix the layout of the product tiles to reflect best practices
Fix variants display on product tile
Removed the uppercase text styling classes from the buttons
Update font variable to be more generic and change the default font for the website to Raleway
Update the font files to use the 5.15.4 version of fontawesome
Remove References to v4 of fontawesome
Update the use of the shopping-cart to the shopping-bag for retail
change Header styling
change Menu Styling
Adjust icon sizes to be smaller
Fix search placeholder styling
Fix search box styling
Update the Home Page banners to use the new responsive advanced versions
Add classes .h1, .h2, .h3, .h4, .h5, .h6, .small to VisualEditor
Add block span to VisualEditor
Add missing headings blocks to VisualEditor

### CODE REVIEW

**Comment by a-rousseau** [comment]
Screen Shot 2022-10-24 at 2 05 46 PM
Screen Shot 2022-10-24 at 2 06 03 PM
Screen Shot 2022-10-24 at 2 06 17 PM
Screen Shot 2022-10-24 at 2 06 33 PM
Screen Shot 2022-10-24 at 2 06 50 PM
Screen Shot 2022-10-24 at 2 07 03 PM
Screen Shot 2022-10-24 at 2 08 31 PM

File: src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_public.xml [hunk 1]
```diff
@@ -43,8 +43,7 @@
...
...tion&gt;&lt;/body&gt;&lt;/html&gt;&lt;/f:param&gt;&lt;/f:function&gt;&lt;/body&gt;&lt;/html&gt;&lt;/f:param&gt;&lt;/f:function&gt;&lt;f:function name=&quot;Composer.SectionTitle&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;Title&quot; value=&quot;This week's specials&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;SubTitle&quot; value=&quot;&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;/f:function&gt;&lt;f:function name=&quot;Composer.Queries.ProductSet.Top&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;QueryName&quot; value=&quot;FeaturedProducts&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;Number&quot; value=&quot;4&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;/f:function&gt;&#xA;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" ChangeDate="2022-10-17 14:22:43Z" ChangedBy="admin" CreationDate="2020-04-28 10:38:50Z" CreatedBy="admin" PublicationStatus="published" SourceCultureName="en-CA" VersionId="f7938305-4286-4173-bbd7-cd91c541dfd1" />
```
Comment (InnaBoitsun) [inline src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_public.xml]: clean VersionId

File: src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_administrated.xml [hunk 2]
```diff
@@ -43,8 +43,7 @@
...
...&gt;&lt;/f:function&gt;&lt;f:function name=&quot;Composer.Queries.ProductSet.Top&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot;&gt;&lt;f:param name=&quot;QueryName&quot; value=&quot;FeaturedProducts&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;f:param name=&quot;Number&quot; value=&quot;4&quot; xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; /&gt;&lt;/f:function&gt;&#xA;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" ChangeDate="2022-10-17 14:22:43Z" ChangedBy="admin" CreationDate="2020-04-28 10:38:50Z" CreatedBy="admin" PublicationStatus="published" SourceCultureName="en-CA" VersionId="f7938305-4286-4173-bbd7-cd91c541dfd1" />  <Add PageId="0ac9d403-ac27-468e-94ac-9f70bfefd063" PlaceHolderId="pageHeader" Content="&lt;html xmlns=&quot;http://www.w3.org/1999/xhtml&quot;&gt;&#xA;&#x9;&lt;head&gt;&#xA;&#x9;&#x9;&lt;f:function xmlns:f=&quot;http://www.composite.net/ns/function/1.0&quot; name=&quot;Composer.Store.PageHeader&quot; /&gt;&#xA;&#x9;&lt;/head&gt;&#xA;&#x9;&lt;body&gt;&#xA;&#x9;&lt;/body&gt;&#xA;&lt;/html&gt;" ChangeDate="2016-02-17 14:51:01Z" ChangedBy="admin" PublicationStatus="published" SourceCultureName="en-CA" />
```
Comment (InnaBoitsun) [inline src/Orckestra.Composer.C1.Content.EN-CA/Package/Datas/Composite.Data.Types.IPagePlaceholderContents_administrated.xml]: clean VersionId

File: src/Orckestra.Composer.Website/App_Data/Razor/Composer/SingleCheckout/OrderSummary.cshtml [hunk 3]
```diff
@@ -17,14 +17,14 @@
...
+            <div class="card">
                 @GeneralHelpers.LoadingSpinner()
 
-                <h3 class="card-header bg-primary border-primary  text-white">
+                <h3 class="card-header card-header-primary">
                     @Html.Localize("ShoppingCart", "T_OrderSummary")
                 </h3>
-                <div class="card-body  bg-light" v-cloak="true">
-                    @OrderSummaryHelpers.Rewards()
+                @OrderSummaryHelpers.Rewards()
```
Comment (InnaBoitsun) [inline src/Orckestra.Composer.Website/App_Data/Razor/Composer/SingleCheckout/OrderSummary.cshtml:26]: you removed v-cloak="true" and now it looks like this:
AZ6pUESZmT
you can add it back inside helper  @OrderSummaryHelpers.Rewards()

Comment (a-rousseau) [reply src/Orckestra.Composer.Website/App_Data/Razor/Composer/SingleCheckout/OrderSummary.cshtml:26]: Oops, will do

**Review by InnaBoitsun — APPROVED** [review]

**Pull request merged**